### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by AdrienBenamira** [eth-sri/mn-bab#1]
Dear all, 

Thank you for this repository and for your work.

Is i possible to provide the corresponding training code as exposed in the paper:

"All three were trained with adversarial training (Madry et al., 2018) using PGD, the GAMA loss (Sriramanan et al., 2020) and MixUp data augmentation (Zhang et al., 2021). ResNet6-A and ResNet8-A were trained using 8-steps and epsilon = 4/255, whereas 20-steps and epsilon = 8/255 were used for ResNet6-B."

Regards
